### PR TITLE
add python-flask-sqlalchemy key to python.yaml

### PR DESCRIPTION
**Pull request by p-panagiotidis**
Please add the following dependency to the rosdep database.

## Package name:

python3-flask-sqlalchemy

## Package Upstream Source:

https://github.com/pallets-eco/flask-sqlalchemy

## Purpose of using this:

Flask-SQLAlchemy is an extension for Flask that adds support for SQLAlchemy

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/bullseye/python3-flask-sqlalchemy
- Ubuntu: https://packages.ubuntu.com/
   - https://packages.ubuntu.com/focal/python3-flask-sqlalchemy
- Fedora: https://packages.fedoraproject.org/
  - https://packages.fedoraproject.org/pkgs/python-flask-sqlalchemy/python3-flask-sqlalchemy/
- Arch: https://www.archlinux.org/packages/
  - https://archlinux.org/packages/community/any/python-flask-sqlalchemy/
- Gentoo: https://packages.gentoo.org/
  - https://packages.gentoo.org/packages/dev-python/flask-sqlalchemy
- macOS: https://formulae.brew.sh/
  - Not available
- Alpine: https://pkgs.alpinelinux.org/packages
  - Not available
- NixOS/nixpkgs: https://search.nixos.org/packages
  - https://search.nixos.org/packages?channel=22.05&show=python39Packages.flask_sqlalchemy&from=0&size=50&sort=relevance&type=packages&query=flask-sqlalchemy
- openSUSE: https://software.opensuse.org/package/
  - https://software.opensuse.org/package/python-Flask-SQLAlchemy

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -1798,6 +1798,15 @@ python-flask-restful:
         packages: [flask-restful]
     xenial: [python-flask-restful]
     yakkety: [python-flask-restful]
+python-flask-sqlalchemy:
```
Comment (clalancette) [inline rosdep/python.yaml:1801]: This should make the Ubuntu package name, so should be:

```suggestion
python3-flask-sqlalchemy:
```

**Review by clalancette — CHANGES_REQUESTED** [review]
One thing to fix, then this is good from my standpoint.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -1798,6 +1798,15 @@ python-flask-restful:
...
     xenial: [python-flask-restful]
     yakkety: [python-flask-restful]
+python3-flask-sqlalchemy:
+  arch: [python-flask-sqlalchemy]
+  debian: [python3-flask-sqlalchemy]
+  fedora: [python3-flask-sqlalchemy]
+  gentoo: [dev-python/flask-sqlalchemy]
+  nixos: [python39Packages.flask_sqlalchemy]
+  ubuntu:
+    '*': [python3-flask-sqlalchemy]
```
Comment (clalancette) [inline rosdep/python.yaml:1808]: Sorry, I just noticed this:

```suggestion
  ubuntu: [python3-flask-sqlalchemy]
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml [hunk 3]
```diff
@@ -1798,6 +1798,14 @@ python-flask-restful:
         packages: [flask-restful]
     xenial: [python-flask-restful]
     yakkety: [python-flask-restful]
+python3-flask-sqlalchemy:
+  arch: [python-flask-sqlalchemy]
+  debian: [python3-flask-sqlalchemy]
+  fedora: [python3-flask-sqlalchemy]
+  gentoo: [dev-python/flask-sqlalchemy]
+  nixos: [python39Packages.flask_sqlalchemy]
```
Comment (clalancette) [inline rosdep/python.yaml:1806]: CI is pointing out that this is available in EPEL-8 for RHEL, which is why it is failing.  This should fix it:

```suggestion
  nixos: [python39Packages.flask_sqlalchemy]
  rhel:
    '*': [python3-flask-sqlalchemy]
    '7': null
```

**Review by clalancette — CHANGES_REQUESTED** [review]
Sorry for all of the back-n-forth here.  I think we are almost there, one more fix for RHEL I think should do it.

File: rosdep/python.yaml [hunk 4]
```diff
@@ -1802,6 +1802,17 @@ python-flatdict-pip:
...
+  rhel:
+    '*': [python3-flask-sqlalchemy]
+    '7': null
+  ubuntu: [python3-flask-sqlalchemy]
+  opensuse: [python-Flask-SQLAlchemy]
```
Comment (clalancette) [inline rosdep/python.yaml:1815]:
```suggestion
  nixos: [python39Packages.flask_sqlalchemy]
  opensuse: [python-Flask-SQLAlchemy]
  rhel:
    '*': [python3-flask-sqlalchemy]
    '7': null
  ubuntu: [python3-flask-sqlalchemy]
```

**Review by clalancette — CHANGES_REQUESTED** [review]
Now CI is complaining about alphabetical order.  Besides the fix I have inline, the entire key should be moved into the `python3-` section of keys in alphabetical order.

**Review by clalancette — APPROVED** [review]

**Review by jacobperron — APPROVED** [review]

**Pull request merged**